app/src/main/java/com/orange/ods/app/ui/components/tabs/MainTabsCustomizationState.kt

```diff
@@ -33,20 +33,20 @@ fun rememberMainTabsCustomizationState(
     bottomSheetScaffoldState: BottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
     tabsCount: MutableIntState,
     pagerState: PagerState = rememberPagerState { tabsCount.intValue.coerceAtLeast(0) },
-    selectedTabsIconPosition: MutableState<OdsTabRow.Tab.Icon.Position> = rememberSaveable { mutableStateOf(OdsTabRow.Tab.Icon.Position.Top) },
+    selectedTabIconPosition: MutableState<OdsTabRow.Tab.Icon.Position> = rememberSaveable { mutableStateOf(OdsTabRow.Tab.Icon.Position.Top) },
```

Comment:
I think we could even rename it to `tabIconPosition` to match the property of `MainTabsCustomizationState`.